community/examples/AMD/README.md

```diff
@@ -0,0 +1,102 @@
+# AMD solutions for the HPC Toolkit
+
+## AMD-Optimized Slurm Cluster
+
+This example provisions a Slurm cluster using the AMD-based Computed Optimized
+"c2d" family of VM types. Additionally, it will install
+[Spack](https://spack.io) and a script to install the AMD Optimizing C/C++ and
+Fortran Compilers ([AOCC](https://developer.amd.com/amd-aocc/)). Installation
+of AOCC requires acceptance of an [End User License Agreement][aocceula] as
+described below.
+
+[aocceula]: https://developer.amd.com/aocc-compiler-eula/
```

Comment:
This site appears valid, but I'm getting the following error when I click on it: 
"An error occurred. Please try again. If you continue to see this error, please contact the web administrator."

This might be temporary, but I wanted to bring it to your attention.

Reply:
No. I think that site may require a referring URL to render. I will swap to a static PDF.

Reply:
Done on HEAD